test/tests/fulltextTest.js

```diff
@@ -5,6 +5,7 @@ describe("Zotero.Fulltext", function () {
 		// Hidden browser, which requires a browser window, needed for charset detection
 		// (until we figure out a better way)
 		win = yield loadBrowserWindow();
+		initPDFToolsPath();
```

Comment:
For some tests it's not enough to initialize `Zotero.Fulltext` paths only in `runtests.js`.